FFL-900 Flags SDK (#1092)
DataDog/dd-sdk-reactnative: packages/react-native-openfeature/README.md

# Datadog OpenFeature Provider for React Native

> Use [OpenFeature][1] with [Datadog Feature Flags][2] to evaluate feature flags and send flag evaluation data to Datadog for monitoring analysis and experimentation.

Reviewer: ```suggestion
Use [OpenFeature][1] with [Datadog Feature Flags][2] to evaluate feature flags and send evaluation data to Datadog to monitor analysis and experimentation.
```

Author: I addressed your feedback in https://github.com/DataDog/dd-sdk-reactnative/pull/1113 (please check this commit: https://github.com/DataDog/dd-sdk-reactnative/pull/1113/changes/c286ddf7e8d553fa5ca7ff43bd9f44fe3b20f37a) because I don't have push rights in this branch

Reviewer: Thank you!

This package provides an OpenFeature-compatible provider that wraps Datadog's Feature Flags SDK.

## What is OpenFeature?

OpenFeature is a vendor-neutral, community-driven specification and SDK for feature flagging. It provides a unified API for feature flag evaluation that works across different providers, so you can switch vendors or integrate multiple feature flag systems.

## Setup

> **Note**: This package is an integration for the [OpenFeature React SDK][1]. Before using it, install and set up the core [`mobile-react-native`][3] SDK.

Reviewer: Is this correct?

```suggestion
**Note**: This package is an integration for the [OpenFeature React SDK][1]. Before using the integration, install and set up the core [`mobile-react-native`][3] SDK.
```

To install with NPM, run:

```sh
npm install @datadog/mobile-react-native @datadog/mobile-react-native-openfeature @openfeature/react-sdk
```

To install with Yarn, run:

```sh
yarn add @datadog/mobile-react-native @datadog/mobile-react-native-openfeature @openfeature/react-sdk
```

## Usage

### Initialize the Datadog SDK and OpenFeature

Use the example code snippet below to initialize the Datadog SDK, enable the Feature Flags feature, and set up the OpenFeature provider.

> After completing this setup, your app should be ready for flag evaluation with OpenFeature.

Reviewer: It seems like this should be moved to after running the code snippet with the following suggested edits:

```suggestion
After completing this setup, your app is ready for flag evaluation with OpenFeature.
```
